lib/options.js

```diff
@@ -115,6 +116,17 @@ class WorkerNodesOptions {
          * @default 100
          */
         this.workerStopTimeout = Number(workerStopTimeout);
+
+        /**
+         * Provides the set of JS engine resource constraints inside this Worker thread.
+         * 
+         * @type {Object}
+         * @property {Number} maxYoungGenerationSizeMb - The maximum size of the main heap in MB
```

Comment:
Same, reversed descriptions in young & old.

Reply:
done ;)